docs/flow/modin/engines/python/pandas_on_python/frame/axis_partition.rst

```diff
@@ -0,0 +1,28 @@
+PandasOnPythonFrameAxisPartition
+""""""""""""""""""""""""""""""""
+
+The class is specific implementation of :py:class:`~modin.engines.base.frame.axis_partition.PandasFrameAxisPartition`,
+providing the API to perform operations on an axis partition, using Python
+as the execution engine. The axis partition is made up of list of block
+partitions that are stored in this class.
+
+Public API
+----------
+
+.. autoclass:: modin.engines.python.pandas_on_python.frame.axis_partition.PandasOnPythonFrameAxisPartition
```

Comment:
```suggestion
.. autoclass:: modin.engines.python.pandas_on_python.frame.axis_partition.PandasOnPythonFrameAxisPartition
  :members:
```
and in other `autoclass` entries, too

Reply:
I don't think we should add `:members:` directive for `PandasOnPythonFrameAxisPartition` autoclass since it will lead to showing of `partition_type` and `instance_type` class attributes, so reverting suggestion.

Reply:
what's wrong with showing the attributes the way other classes are documented?..

Reply:
We have [decided](https://github.com/modin-project/modin/pull/3067#discussion_r638714452) to hide these attributes)

Reply:
From the docstring - yes, but not from the documentation page :) Those are different things.

Reply:
Ok, we can add `:members:` directive here, but in this case (without using `:exclude-members:` directive and explicit addition of attributes description in the docstring), `Public API` will not contain any description and looks as follows
![image](https://user-images.githubusercontent.com/55585026/119661627-264e2180-be39-11eb-8bcc-be803f686cd6.png)